mysqlchk.go

```diff
@@ -43,7 +47,25 @@ func checkHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	err := wsrepStmt.QueryRow().Scan(&fieldName, &wsrepState)
+	err := maxConnStmt.QueryRow().Scan(&fieldName, &maxConn)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	err = connUsedStmt.QueryRow().Scan(&fieldName, &connUsed)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	if maxConn <= (connUsed + 10) {
```

Comment:
Let's add runtime flags for this feature. Either one to enable connection count checking and another to set the limit (10 by default), or just a limit (0 by default, meaning the check is disabled).